x-pack/plugins/ingest_pipelines/public/application/sections/pipelines_list/details.tsx

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FunctionComponent } from 'react';
+import { i18n } from '@kbn/i18n';
+import {
+  EuiFlyout,
+  EuiFlyoutHeader,
+  EuiFlyoutBody,
+  EuiTitle,
+  EuiDescriptionList,
+  EuiCodeBlock,
+  EuiFlyoutFooter,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiButtonEmpty,
+} from '@elastic/eui';
+import { Pipeline } from '../../../../common/types';
+
+export interface Props {
+  pipeline: Pipeline;
+  onEditClick: () => void;
+  onDeleteClick: () => void;
+  onClose: () => void;
+}
+
+export const PipelineDetails: FunctionComponent<Props> = ({
+  pipeline,
+  onClose,
+  onEditClick,
+  onDeleteClick,
+}) => {
+  const descriptionListItems = [
+    {
+      title: i18n.translate('xpack.ingestPipelines.list.pipelineDetails.descriptionTitle', {
+        defaultMessage: 'Description',
+      }),
+      description: pipeline.description ?? '',
+    },
+  ];
+
+  if (pipeline.version) {
+    descriptionListItems.push({
+      title: i18n.translate('xpack.ingestPipelines.list.pipelineDetails.versionTitle', {
+        defaultMessage: 'Version',
+      }),
+      description: String(pipeline.version),
+    });
+  }
+
+  descriptionListItems.push({
+    title: i18n.translate('xpack.ingestPipelines.list.pipelineDetails.processorsTitle', {
+      defaultMessage: 'Processors JSON',
+    }),
+    // We use this title from the description list and display the processors in a code block underneath
```

Comment:
This seems a little odd to me, as the code block ends up rendering outside the description list (`<dl>`). It might cause accessibility issues.

<img width="609" alt="Screen Shot 2020-04-08 at 11 12 07 AM" src="https://user-images.githubusercontent.com/5226211/78800808-e3599200-7989-11ea-9d9b-05937645e7c4.png">